app/services/account_reset/track_irs_event.rb

```diff
@@ -0,0 +1,29 @@
+module AccountReset::TrackIrsEvent
```

Comment:
This is a mixin, and looking around it seems to have an implicit dependency on `sp` and maybe `success`, are ther any others? I would write them in a comment at the top of the file
```suggestion
# Mixin for account reset event tracking
# Assumes these methods exist on the including class:
# - sp
# - success
# ...
module AccountReset::TrackIrsEvent
```

Reply:
Yes, :success, :errors, :request, :analytics

Reply:
also can we call it `success?` since it's a boolean?